index.js

```
@@ -1,57 +1,59 @@
-'use strict';
+'use strict'
 
-// External Modules
-const EventEmitter = require('events').EventEmitter;
-const Logger = require('logdna').Logger;
+const {EventEmitter} = require('events')
+const {createLogger} = require('@logdna/logger')
+const pkg = require('./package.json')
 
 // Constants
 const levels = {
-    10: 'TRACE'
-    , 20: 'DEBUG'
-    , 30: 'INFO'
-    , 40: 'WARN'
-    , 50: 'ERROR'
-    , 60: 'FATAL'
-};
+  10: 'TRACE'
+, 20: 'DEBUG'
+, 30: 'INFO'
+, 40: 'WARN'
+, 50: 'ERROR'
+, 60: 'FATAL'
```

do we know if this is a complete list? is there any scenario where we might get an integer that is NOT one of these? currently this would fail hard below if we do.

---

Great question, but thankfully yes, these are all the levels in bunyan and they match our node logger perfectly. https://github.com/trentm/node-bunyan#levels